appuselfbot.py

```diff
@@ -106,24 +109,25 @@ def Wizard():
         Wizard()
 
 shutdown = False
-if not get_config_value('config', 'run_as_superuser'):
+if os.name == 'nt':
+    try:
+        # only windows users with admin privileges can read the C:\windows\temp
+        temp = os.listdir(os.sep.join([os.environ.get('SystemRoot','C:\\windows'),'temp']))
+    except:
+        shutdown = False
+    else:
+        shutdown = True
+else:
+    if 'SUDO_USER' in os.environ and os.geteuid() == 0:

```

Comment:
That is, running only as root user without sudo? The SUDO_ environment variables should exist only when sudo is invoked.